index.js

```diff
@@ -167,7 +167,7 @@ function mergeParams (...params) {
 }
 
 function addRouteRateHook (pluginComponent, params, routeOptions) {
-  const hook = params.hook || defaultHook
+  const hook = params.hook
```

Comment:
It is already [set in globalParams](https://github.com/fastify/fastify-rate-limit/blob/595a627020837d5c4ad3de7a235f385f3e6af2a5/index.js#L73) and [merged with custom params](https://github.com/fastify/fastify-rate-limit/blob/595a627020837d5c4ad3de7a235f385f3e6af2a5/index.js#L130)